README.md

```diff
@@ -1,7 +1,5 @@
 # AEM Groovy Console
```

Comment:
maybe we can also add badges in the future for the build and maven release ;)

Reply:
definitely!